Applications/spire/source/common/local_technicals_model.cpp

```diff
@@ -6,37 +6,54 @@ using namespace Nexus;
 using namespace Spire;
 
 LocalTechnicalsModel::LocalTechnicalsModel(Security security)
-    : m_security(security) {}
+    : m_security(std::move(security)),
+      m_volume(0),
+      m_high_changed(false),
+      m_low_changed(false),
+      m_open_changed(false),
+      m_last_price_changed(false) {
+  m_volume_signal(m_volume);
```

Comment:
Who will be receiving this signal?

Reply:
So should this be emitted on load then?

Reply:
The volume signal is emitted when the volume is updated.

Reply:
So the default value doesn't count as updating it?

Reply:
Sure you can consider that, but then ask yourself who/what will be receiving the signal in the constructor? How can it be possible that someone will receive a signal before an object has been constructed?

Reply:
I know technically it's just like yelling into a void - there's no one there to receive it.

But if I can't signal the volume on construction then it's either I can emit it when the model loads or it may be the case that a signal isn't emitted at all when the volume is 0.

Reply:
Don't emit anything.